### PR TITLE
updating authority field

### PR DESCRIPTION
**Pull request by Paulmulonzia**

### CODE REVIEW

File: lib/collectionspace/converter/publicart/config.yml [hunk 1]
```diff
@@ -129,31 +129,45 @@ registered_profiles:
...
+      Authorities:
+        - name_field: currentplace
+          authority_type: Place
+          authority_subtype: place
+        - name_field: foundingplacelocal
+          authority_type: Place
+          authority_subtype: place
+        - name_field: foundingplaceshared
+          authority_type: Place
+          authority_subtype: place
```
Comment (kspurgin) [inline lib/collectionspace/converter/publicart/config.yml:152]: @Paulmulonzia - the authority_subtype needs to be the `shortIdentifier` for the given authority. I usually look this up using the API: https://publicart.dev.collectionspace.org/cspace-services/placeauthorities  

It is also found in the refName URN of a term in the authority, in parentheses after `{*}authorities:name`

so this should be `place_shared`

**Comment by kspurgin** [comment]
@Paulmulonzia - also, if you are retrospectively going back and updating configs for existing modules, you can do all those updates in one branch/pull request. That'll make it easier to review and manage them.

**Comment by Paulmulonzia** [comment]
@kspurgin Ready for review

**Comment by kspurgin** [comment]
@mark-cooper - this is ready for merge into master

**Pull request merged**